packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -292,16 +293,22 @@ const makeRenderer = (
 		});
 	});
 
-	return initializeStore.then(initializedStore =>
-		getRouterRenderer({
-			routes,
-			store: initializedStore,
-			location: url,
-			basename,
-			scripts,
-			cssLinks,
-			userAgent,
-		})
+	return initializeStore.then(
+		initializedStore =>
+			// create tracer and immediately invoke the resulting function
+			// trace should start before rendering, finish after rendering
+			newrelic.createTracer(
+				'serverRender',
+				getRouterRenderer({
+					routes,
+					store: initializedStore,
+					location: url,
+					basename,
+					scripts,
+					cssLinks,
+					userAgent,
+				})
+			)() // self-invoke
```

Comment:
Why are we invoking here but in the other parts of this PR we are not?

Reply:
good question! I've outlined what I _think_ will happen in the comments above where `createTracer` is called, and I'm eager to see how this works in practice, but I think this is the correct way of putting a timer around a synchronous function call.

`createTracer` is designed primarily for async callback-based patterns, but in this case I'm expecting the synchronous, blocking function to take a non-trivial amount of time, so the 'callback' is called immediately. According to the docs, the timer for `createTracer` starts when `createTracer` is first called and finishes _after the callback returns_, so this should effectively measure the execution time of the callback itself.

1. call `createTracer`
2. invoke callback
3. callback complete